dom/data/data.js

```diff
@@ -1,14 +1,23 @@
-var isEmptyObject = require("../../js/is-empty-object/is-empty-object");
-var data = {};
-var expando = 'can'+new Date();
-var uuid = 0;
-var setData = function(name, value) {
-	var id = this[expando] || (this[expando] = ++uuid),
-		store = data[id] || (data[id] = {});
-	if (name !== undefined) {
-		store[name] = value;
+var domDataCore = require("./core");
+var mutationDocument = require("../mutation-observer/document/document");
+
+var deleteNode = function() {
+	return domDataCore.delete.call(this);
+};
+
+// keep count of how many times setData was called
+var keysSetCount = 0;
+
+var cleanupDomData = function(node) {
+	var deletedKeys = deleteNode.call(node);
```

Comment:
maybe instead of the count, maybe it should just let us know if there's anything else to delete.